pallets/gear/src/manager.rs

```diff
@@ -382,17 +379,19 @@ where
                 let _ = T::GasHandler::split(message_id, dispatch.id().into_origin());
             }
 
-            <MessengerPallet<T> as Messenger>::Queue::push_back(dispatch)
+            QueueOf::<T>::queue(dispatch)
                 .unwrap_or_else(|e| unreachable!("Message queue corrupted! {:?}", e));
         } else {
             let message = dispatch.into_parts().1;
 
             // Being placed into a user's mailbox means the end of a message life cycle.
             // There can be no further processing whatsoever, hence any gas attempted to be
             // passed along must be returned (i.e. remain in the parent message's value tree).
-            Pallet::<T>::insert_to_mailbox(message.destination().into_origin(), message.clone());
-
-            Pallet::<T>::deposit_event(Event::Log(message));
+            if MailboxOf::<T>::insert(message.clone()).is_ok() {
+                Pallet::<T>::deposit_event(Event::Log(message));
+            } else {
+                log::debug!("Error occured in mailbox insertation")
```

Comment:
```suggestion
                log::error!("Error occured in mailbox insertation")
```

Reply:
Disagree here, because, for some reason, we use everywhere log::debug for any errors occurred.

Reply:
are there any special reasons for that?

Reply:
> are there any special reasons for that?

No idea :/

Reply:
Probably for comfortable logs enabling by `RUST_LOG=debug`?

Reply:
But `error` logs is included in `debug` mode.